src/storhaug

```diff
@@ -15,7 +15,7 @@ HA_SERVERS=""
 SYS_CONFDIR="/etc/sysconfig"
 HA_CONF="${SYS_CONFDIR}/storhaug.conf"
 HA_CONF_INCDIR="${SYS_CONFDIR}/storhaug.d"
-HA_MNT_DIR="/run/storhaug"
+HA_MNT_DIR="/run/gluster"
```

Comment:
This change seems unnecessary and I don't like that it ties storhaug more closely with Gluster. The idea is that eventually other storage solutions could be used as well.